brave/common/extensions/url_bindings.cc

```diff
@@ -300,6 +304,13 @@ void URLBindings::New(const v8::FunctionCallbackInfo<v8::Value>& args) {
 
 void URLBindings::FormatForDisplay(
     const v8::FunctionCallbackInfo<v8::Value>& args) {
+  this->FormatForDisplay(args, net::UnescapeRule::NONE);
+}
+
+void URLBindings::FormatForDisplay(
```

Comment:
There's no reason to have a second second method signature. `args` can handle any number of args and can also handle optional args